### DISCUSSION TITLE
FeedIterator.HasMoreResults returns false in azure hosting

### DISCUSSION THREAD

**Discussion by Daniel-Guenter** [Azure/azure-cosmos-dotnet-v3#2095] [Q&A]
I am trying to get the pull model of the change feed working as described here.

https://docs.microsoft.com/en-us/azure/cosmos-db/change-feed-pull-model

```
                using (var feedIterator = _container.GetChangeFeedIterator<NotificationRequestItemDbContainer>(ChangeFeedStartFrom.Time(startUtc)))
                {
                    while (feedIterator.HasMoreResults)
                    {
                        cancellationToken.ThrowIfCancellationRequested();

                        try
                        {
                            var matchedChanges = (await feedIterator.ReadNextAsync(cancellationToken))
```

I am using version 3.15.2-preview of the .net sdk.  When I use this code against my local emulator everything works fine.  HasMoreResults is always true as described in the documentation.  However when I use a cosmos DB hosted in azure, the HasMoreResults returns false immediately.  Even I loop and wait for HasMoreResults to be true it will never return any results.  I know the connection and the container are fine because I can query the data fine with a GetItemQueryIterator. 

Am I missing some setup to get the change feed to work in azure?

**Comment by j82w** [comment]
Everything looks correct. This seems like a bug. Does this reproduce if you use a new container? Can you create an issue for this?

**Reply by Daniel-Guenter** [reply]
Thanks for the reply.  I did try with a new container with the same results.  I will create an issue for this.
